src/mds/nameserver2/curvefs.cpp

```diff
@@ -1251,9 +1258,15 @@ StatusCode CurveFS::GetOrAllocateSegment(const std::string & filename,
             return  StatusCode::kSegmentNotAllocated;
         } else {
             // TODO(hzsunjianliang): check the user and define the logical pool
+            if (!fileInfo.has_poolsetname()) {
+                fileInfo.set_poolsetname("ssdPoolset1");
```

Comment:
it's no a good idea, a proper way is adding a config item in mds configuration to store the default poolsetname for allocating segment for those files that don't have a poolsetname in fileinfo.

Reply:
> it's no a good idea, a proper way is adding a config item in mds configuration to store the default poolsetname for allocating segment for those files that don't have a poolsetname in fileinfo.

fix, defalut poolsetName has been set in mds.conf